modules/ROOT/pages/asm-secret-group-concept.adoc

```diff
@@ -5,33 +5,40 @@ endif::[]
 
 A secret group is a unique secrets vault to which all your secrets are bound. Each secret group has individual access and authorization controls.
 
-Secrets Manager stores your secrets per secret group, per environment, and per business group. So, only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
+Secrets Manager stores your secrets per secret group, per environment and per business group. Only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
 
-To ensure that all clients use a consistent set of secrets and that all users edit the same set of secrets at a given time, Secrets Manager provides both high availability and consistency of stored data, as well as concurrent use functionality for secret renewals.
+Secrets Manager provides both high availability and consistency of stored data, as well as concurrent use functionality for secret renewals. This ensures that all clients use a consistent set of secrets and all users edit the same set of secrets at a given time. 
 
 == Availability and Consistency
 
-By implementing Edit Sessions every time a user needs to edit secrets inside a secret group, Secrets Manager guarantees service of a consistent set of secrets.
+Secrets Manager implements edit sessions every time a user needs to edit secrets inside a secret group to guarantee service of a consistent set of secrets.
 
-Consider the scenario where you need to renew secrets inside a secret group: You need to click the Edit button in the Anypoint Platform UI. This opens an edit session during which the secret group locks out other users, and you can edit all the necessary security objects inside the secret group. While the edit session is open, Secrets Manager serves the same secrets it had before you opened the edit session. This guarantees that any redeployed application would still see the same, consistent secret group even though you are modifying it.
+For example, if you need to renew secrets inside a secret group, you click the *Edit* button in the Secrets Manager UI. This opens an edit session and locks out other users while you edit the necessary security objects inside the secret group. 
```

Comment:
I'd start with an intro statement first to set context, something like "Secrets Manager ensures that operations are not interrupted while you are modifying secrets. When you edit a secrets group, other users are locked out while you..."